decentraland/unity-explorer
# Fix: double disposale in Livekit pooling

## What does this PR change?

ReceivedMessage or any struct with disposable shouldn't be passed in multiple places, because the internal state is copying and those copies are not synchronised between each other and could be easily used unproperly 

`SceneMessage` is now fully immutable and couldn't be mutated and couldn't be used unproperly

## How to test the changes?

1. Launch the explorer and bring your peeps together
2. Go to /world nicoe
3. Run around and see no errors
4. Go to /goto 0,0
5. No errors should occur

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
